### ISSUE TITLE
Add CSS clamp to Customizer h1-h6

### ISSUE THREAD

**Issue by macbaydigital** [tomusborne/generatepress#426]
### Description
Describe your feature request.

In the customizer, I'd love to be able to apply fluid responsive font-sizing of headings via CSS **clamp**

**Comment by d8m18n** [comment]
+1 Yes, it would be great to set a min and max size and let  CSS calculate the in between.

**Comment by thisbit** [comment]
Yes but not only to Hs, to typography section in the customizer in general. Or at leastlet us define them as variables and use in customizer ... something!!!

**Comment by 3xploiton3** [comment]
i need this 

+1

reference: https://coreframework.com/app

**Comment by rsoeldner** [comment]
> Yes but not only to Hs, to typography section in the customizer in general. Or at leastlet us define them as variables and use in customizer ... something!!!

Isn’t this already the case? Using a child theme specifying the clamp vars and referencing them inside the customizer (and GB global styles)?

**Comment by diggeddy** [comment]
Closing - typograpghy fields in customiser support string values including clamp property.
